datafusion/sqllogictest/bin/sqllogictests.rs

```diff
@@ -62,6 +62,11 @@ async fn run_tests() -> Result<()> {
     env_logger::init();
 
     let options: Options = clap::Parser::parse();
+    // nextest parses stdout
+    if options.list {
+        eprintln!("NOTICE: --list option unsupported quitting");
+        return Ok(());
```

Comment:
why Ok?

Reply:
Because if it returns an error, `cargo nextest` aborts immediately

What is happening I think is that `nextest` lists all possible commands first (by running `cargo test ... --list --format terse`) and then uses that output to determine what is subsequently run. 

So if the `--list` part returns an error then none of the tests run

Reply:
Added some comments in fd1552366